### PR TITLE
ticdc: Update migrate from tidb to tidb doc

### PR DESCRIPTION
**Pull request by sdojjy**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: migrate-from-tidb-to-tidb.md [hunk 1]
```diff
@@ -144,6 +144,10 @@ aliases: ['/zh/tidb/dev/incremental-replication-between-clusters/']
     1 row in set (0.00 sec)
     ```
 
+> **注意：**
+>
+> 默认的 `gc-ttl` 为 24 小时可能无法满足需求。此时应该根据实际情况，在启动 TiCDC server 时指定 `gc-ttl` 的值。请参考:[TiCDC 的 `gc-ttl` 是什么](/ticdc/ticdc-faq.md#ticdc-的-gc-ttl-是什么)
```
Comment (qiancai) [inline migrate-from-tidb-to-tidb.md:149]: 这个注意提到的是 “在启动 TiCDC server 时指定 `gc-ttl` 的值”，但是 L149 这里还没开始部署 TiCDC，需要把这个注意挪到下面这里吗

image

Comment (qiancai) [inline migrate-from-tidb-to-tidb.md:149]:
```suggestion
> TiCDC 的 `gc-ttl` 默认为 24 小时。如果此时长无法满足需求，请在启动 TiCDC server 时根据实际情况指定 `gc-ttl` 的值。更多信息，请参考 [TiCDC 的 `gc-ttl` 是什么](/ticdc/ticdc-faq.md#ticdc-的-gc-ttl-是什么)。
```

File: migrate-from-tidb-to-tidb.md [hunk 2]
```diff
@@ -144,6 +144,10 @@ aliases: ['/zh/tidb/dev/incremental-replication-between-clusters/']
     1 row in set (0.00 sec)
     ```
 
+> **注意：**
+>
+> TiCDC 的 `gc-ttl` 默认为 24 小时。如果此时长无法满足需求，请在启动 TiCDC server 时根据实际情况指定 `gc-ttl` 的值。更多信息，请参考 [TiCDC 的 `gc-ttl` 是什么](/ticdc/ticdc-faq.md#ticdc-的-gc-ttl-是什么)。
```
Comment (sdojjy) [inline migrate-from-tidb-to-tidb.md:149]:
```suggestion
> TiCDC 的 `gc-ttl` 默认为 24 小时, 如果备份恢复耗时过长，此时  `gc-ttl` 可能无法满足需求， 导致同步任务运行失败，请在启动 TiCDC server 时根据实际情况指定 `gc-ttl` 的值。更多信息，请参考 [TiCDC 的 `gc-ttl` 是什么](/ticdc/ticdc-faq.md#ticdc-的-gc-ttl-是什么)。
```

File: migrate-from-tidb-to-tidb.md [hunk 3]
```diff
@@ -144,6 +144,10 @@ aliases: ['/zh/tidb/dev/incremental-replication-between-clusters/']
     1 row in set (0.00 sec)
     ```
 
+> **注意：**
+>
+> TiCDC 的 `gc-ttl` 默认为 24 小时, 如果备份恢复耗时过长，此时  `gc-ttl` 可能无法满足需求， 导致同步任务运行失败，请在启动 TiCDC server 时根据实际情况指定 `gc-ttl` 的值。更多信息，请参考 [TiCDC 的 `gc-ttl` 是什么](/ticdc/ticdc-faq.md#ticdc-的-gc-ttl-是什么)。
```
Comment (qiancai) [inline migrate-from-tidb-to-tidb.md:149]:
```suggestion
> TiCDC 的 `gc-ttl` 默认为 24 小时。如果备份恢复耗时过长，默认的 `gc-ttl` 可能无法满足需求，导致同步任务运行失败。请在启动 TiCDC server 时根据实际情况指定 `gc-ttl` 的值。更多信息，请参考 [TiCDC 的 `gc-ttl` 是什么](/ticdc/ticdc-faq.md#ticdc-的-gc-ttl-是什么)。
```

File: migrate-from-tidb-to-tidb.md [hunk 4]
```diff
@@ -144,6 +144,10 @@ aliases: ['/zh/tidb/dev/incremental-replication-between-clusters/']
     1 row in set (0.00 sec)
     ```
 
+> **注意：**
+>
+> TiCDC 的 `gc-ttl` 默认为 24 小时。如果备份恢复耗时过长，默认的 `gc-ttl` 可能无法满足需求，从而导致后续的[增量同步任务](/update-migrate-from-tidb-to-tidb/migrate-from-tidb-to-tidb.md#第-3-步迁移增量数据)运行失败。为了避免这种情况，请在启动 TiCDC server 时根据实际需求调整 `gc-ttl` 的值。更多信息，请参考 [TiCDC 的 `gc-ttl` 是什么](/ticdc/ticdc-faq.md#ticdc-的-gc-ttl-是什么)。
```
Comment (qiancai) [inline migrate-from-tidb-to-tidb.md:149]:
```suggestion
    > **注意：**
    >
    > TiCDC 的 `gc-ttl` 默认为 24 小时。如果备份恢复耗时过长，默认的 `gc-ttl` 可能无法满足需求，从而导致后续的[增量同步任务](/update-migrate-from-tidb-to-tidb/migrate-from-tidb-to-tidb.md#第-3-步迁移增量数据)运行失败。为了避免这种情况，请在启动 TiCDC server 时根据实际需求调整 `gc-ttl` 的值。更多信息，请参考 [TiCDC 的 `gc-ttl` 是什么](/ticdc/ticdc-faq.md#ticdc-的-gc-ttl-是什么)。
```

**Review by 3AceShowHand — APPROVED** [review]

**Review by flowbehappy — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

File: migrate-from-tidb-to-tidb.md [hunk 5]
```diff
@@ -144,6 +144,10 @@ aliases: ['/zh/tidb/dev/incremental-replication-between-clusters/']
     1 row in set (0.00 sec)
     ```
 
+    > **注意：**
+    >
+    > TiCDC 的 `gc-ttl` 默认为 24 小时。如果备份恢复耗时过长，默认的 `gc-ttl` 可能无法满足需求，从而导致后续的[增量同步任务](/update-migrate-from-tidb-to-tidb/migrate-from-tidb-to-tidb.md#第-3-步迁移增量数据)运行失败。为了避免这种情况，请在启动 TiCDC server 时根据实际需求调整 `gc-ttl` 的值。更多信息，请参考 [TiCDC 的 `gc-ttl` 是什么](/ticdc/ticdc-faq.md#ticdc-的-gc-ttl-是什么)。
```
Comment (qiancai) [inline migrate-from-tidb-to-tidb.md:149]:
```suggestion
    > TiCDC 的 `gc-ttl` 默认为 24 小时。如果备份恢复耗时过长，默认的 `gc-ttl` 可能无法满足需求，从而导致后续的[增量同步任务](#第-3-步迁移增量数据)运行失败。为了避免这种情况，请在启动 TiCDC server 时根据实际需求调整 `gc-ttl` 的值。更多信息，请参考 [TiCDC 的 `gc-ttl` 是什么](/ticdc/ticdc-faq.md#ticdc-的-gc-ttl-是什么)。
```

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**